Wikipedia/Categories/UITableViewCell+WMFLayout.m

```diff
@@ -0,0 +1,15 @@
+//  Created by Monte Hurd on 11/23/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import "UITableViewCell+WMFLayout.h"
+#import "NSProcessInfo+WMFOperatingSystemVersionChecks.h"
+
+@implementation UITableViewCell (WMFLayout)
+
+- (void)wmf_layoutIfNeededIfOperatingSystemVersionLessThan9_0_0 {
+    if ([[NSProcessInfo processInfo] wmf_isOperatingSystemVersionLessThan9_0_0]) {
+        [self layoutIfNeeded];
```

Comment:
Can't we just call `layoutIfNeeded` always? Or to ask another way, what is the downside of calling `layoutIfNeeded` in iOS 9?